util/types/time.go

```diff
@@ -437,6 +437,15 @@ func (t *Time) check() error {
 	return nil
 }
 
+// TotalDays returns number of days starting from 0/1/1
+func (t Time) TotalDays() int {
```

Comment:
Guys don't you think that convertDatumToTime should check if date has value like 2016-00-13 and return parsing error in that case? or in another place, before builtin method is called?